src/lib/utils/crypto.ts

```diff
@@ -0,0 +1,39 @@
+import { reduce } from 'lodash';
+import { isAddress, keccak256, toHex } from 'web3-utils';
+import { publicToAddress } from 'ethereumjs-util';
+import { ETH_ZERO_ADDR, ETH_ZERO_ADDR_SHORT } from 'lib/constants';
+
+export const stripHexPrefix = (hex: string) => {
+  return hex.startsWith('0x') ? hex.slice(2) : hex;
+};
+
+export const addHexPrefix = (hex: string) => {
```

Comment:
These are all just hex/Ethereum address functions, so the `crypto.ts` name feels a bit misleading. Perhaps `utils/address.ts`?